### DISCUSSION TITLE
gphoto2 parameter

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#225] [Q&A] [answered]
In the config you offer lots off backend parameter for the webcam but only a view for gphoto2 .
Is this because not all cameras support all parameters that are offert by gphoto2 ?

**Comment by mgineer85** [comment]
Yes, only supported settings are added.
What you miss?

**Reply by Striker62** [reply, marked as answer]
I've cheched what my camera offers .
Only capture ;-)

my idea was to set size and quality parameter, but these are only as read only provided by my nikon d3100
So I'good with the gphoto2 config
